### ISSUE TITLE
Navigation bar not controlled by Apex Launcher Classic

### ISSUE THREAD

**Issue by AM4Ryzen** [hockeymikey/android_device_sony_lilac#64]
There's a option in Apex Launcher Classic where it makes the navbar and status bar opaque, however on this ROM, it only effects the status bar.

![Screenshot_20210725-120039_Apex_Launcher_Classic](https://user-images.githubusercontent.com/39643236/126905565-dc2ef288-d5f0-4368-99c4-0153474b63c8.png)

**Comment by hockeymikey** [comment]
Does it work in other roms? I'll have to look into how they are trying to achieve the navbar change.
https://medium.com/@jayshildaveofficial/android-translucent-navigation-bar-7b21ddff7e43
https://stackoverflow.com/questions/29069070/completely-transparent-status-bar-and-navigation-bar-on-lollipop

**Comment by AM4Ryzen** [comment]
> Does it work in other roms? I'll have to look into how they are trying to achieve the navbar change.
> https://medium.com/@jayshildaveofficial/android-translucent-navigation-bar-7b21ddff7e43
> https://stackoverflow.com/questions/29069070/completely-transparent-status-bar-and-navigation-bar-on-lollipop

Yes, I've seen it work on LineageOS 15.1 and 14.1

Also, this could be related
![Screenshot_20210725-145742_DiscordK](https://user-images.githubusercontent.com/39643236/126910340-b152a399-875a-4a58-8cb2-ee140792521d.png)
(Accidently posted this under wrong issue first time)

**Comment by hockeymikey** [comment]
Hmm, so it's bugged throughout when Apex tries to change it?

**Comment by AM4Ryzen** [comment]
> Hmm, so it's bugged throughout when Apex tries to change it?

Well, no. I just think RR is too controlling over the navigation bar. If there was an option to relinquish RR's control over the navigation bar and make it the stock LineageOS navigation bar, it would work fine.

**Comment by hockeymikey** [comment]
> > Hmm, so it's bugged throughout when Apex tries to change it?
> 
> Well, no. I just think RR is too controlling over the navigation bar. If there was an option to relinquish RR's control over the navigation bar and make it the stock LineageOS navigation bar, it would work fine.

That's kinda more work. Honestly just getting it to work with Apex and exploring that would be the ticket.

**Comment by AM4Ryzen** [comment]
> That's kinda more work. Honestly just getting it to work with Apex and exploring that would be the ticket.

Actually, the RR navigation bar appears to also effect the Sony Camera app. Decrease the navigation bar size, and the buttons will come partially into view.

**Comment by hockeymikey** [comment]
So the RR navigation bar is too big compared to stock?

**Comment by AM4Ryzen** [comment]
No, it's about the same size, apps just can't control it unlike the stock/Lineage navbar. The camera app usually hides the navbar, letting it use the full space.

**Comment by hockeymikey** [comment]
Interesting observations. Apps usually just go full screen but

**Comment by AM4Ryzen** [comment]
I've also found that some GravityBox modifications, changing size for example, do nothing. So I think there needs to be like a stock navbar option without any customizations or just the stock LineageOS ones for it to work.
